### ISSUE TITLE
Heroic Wartorn Rathrak, The Poisonous Mind, does not work with metamorphesis damage increase

### ISSUE THREAD

**Issue by Campusboard123** [gamefreedomgit/Maelstrom#5685]
[//]: # (REMBEMBER! Add links to things related to the bug using for example:)
[//]: # (http://wowhead.com/)
[//]: # (cata-twinhead.twinstar.cz)

**Description:**
Hello. I play metamorph warlock, essentially when I use that spell it increases damage by 60% when I use it, and all tooltip change accondingly (example: cunning of the cruel trinket)
The tooltip also changes for Wartorn Rathrak (heroic), but the dot damage is the same, and the spell debuff on enemies is the same.
TLDR: wether I am in meta (60% more damage) or not, the wepon (despite its tooltip changing) DOES THE SAME DAMAGE, regardless of the 60% increase IT SHOULD DO.
Cunning of the cruel works as intended, in comparison.
**How to reproduce:**

**How it should work:**

**Database links:**

**Comment by Sketish** [comment]
Provide any retail proof it should scale

"This weapon's damage does NOT scale with anything other than spell damage and nature spell damage modifiers.
Even the tooltip on weapon changes, the tooltip on buff ([ Blast of Corruption](https://www.wowhead.com/spell=107831/blast-of-corruption)) always stays as "1742 damage...", BUT the real damage it does is what weapon tooltip says plus any spell damage modifier applied on mob (e.g: [(Spell #48506)](https://www.wowhead.com/spell=48506)).
Since [(Spell #77492)](https://www.wowhead.com/spell=77492) increases damage of your nature spells within[ Eclipse (Solar)](https://www.wowhead.com/spell=48517/eclipse-solar), it also increases this weapon's damage and this is shown in weapon's tooltip.

**Comment by Revly** [comment]
> Since [(Spell #77492)](https://www.wowhead.com/spell=77492) increases damage of your nature spells within[ Eclipse (Solar)](https://www.wowhead.com/spell=48517/eclipse-solar), it also increases this weapon's damage and this is shown in weapon's tooltip.

It also increases with [(Elemental Mastery)](cata-twinhead.twinstar.cz/?spell=16166) I suppose?
I have not tested

**Comment by Pornoodemon** [comment]
> Provide any retail proof it should scale
> 
> "This weapon's damage does NOT scale with anything other than spell damage and nature spell damage modifiers. Even the tooltip on weapon changes, the tooltip on buff ([ Blast of Corruption](https://www.wowhead.com/spell=107831/blast-of-corruption)) always stays as "1742 damage...", BUT the real damage it does is what weapon tooltip says plus any spell damage modifier applied on mob (e.g: [(Spell #48506)](https://www.wowhead.com/spell=48506)). Since [(Spell #77492)](https://www.wowhead.com/spell=77492) increases damage of your nature spells within[ Eclipse (Solar)](https://www.wowhead.com/spell=48517/eclipse-solar), it also increases this weapon's damage and this is shown in weapon's tooltip.

It does not scale with characteristics.
Any damage modifier (that will increase overall magic damage) should increase the damage of that weapon.
The difference between Eclipse (Solar) and Metamorphosis is that Metamorphosis increases damage for all schools of magic.
And this weapon cannot increase damage for only one specialization in the game

**Comment by Pornoodemon** [comment]
@Campusboard123 
![srewrewq](https://github.com/gamefreedomgit/Maelstrom/assets/133828891/e40d4aff-6711-43f3-b206-d8010f0215eb)
Everything works as it should work.
And specifically in your case, it didn’t work, most likely because you used metamorphosis after proc DOT.
Use metamorphose and then get the Proc and you will see an increase in damage due to the fact that all DOT take a Snapshot at the moment of use
@Sketish 
This works similarly on Twin.
![image](https://github.com/gamefreedomgit/Maelstrom/assets/133828891/8ff6e969-c2cd-4afc-aeee-252bb12c53a7)
